backend/orchestrator/common_pb2.pyi

```diff
@@ -68,6 +68,7 @@ ASCENDING: SortOrder.ValueType  # 1
 DESCENDING: SortOrder.ValueType  # 2
 global___SortOrder = SortOrder
 
+@typing_extensions.final
```

Comment:
Out of curiosity, what is this line doing?

Reply:
Oh, nice https://peps.python.org/pep-0591/#the-final-decorator

Reply:
It was auto generated, I didn't have your curiosity ahah. Thanks for pointing that out, really interesting indeed !